docs/examples/201/web-app-conditional-log/main.bicep

```diff
@@ -0,0 +1,109 @@
+param skuName string = 'S1'

```

Comment:
looks like web-app-conditional-log got include in this PR by mistake

Reply:
Good catch, missed that on branch management.

Should be resolved now.